charts/addresspools/templates/ipaddresspool-default.yaml

```
@@ -1,10 +1,10 @@
-{{- if .Values.ipAddressPool }}
+{{- if .Values.ipAddressPoolCIDR }}
```

This doesn't match the default values you have set.
```suggestion
{{- if .Values.default.ipAddressPoolCIDR }}
```